src/components/NavbarDrawer.js

```diff
@@ -52,10 +54,66 @@ const useStyles = makeStyles((theme) => ({
 function NavbarDrawer({ updateTT, toggleTheme }) {
   const classes = useStyles();
   const [mobileOpen, setMobileOpen] = React.useState(false);
-  const handleDrawerToggle = () => {
+  function handleDrawerToggle() {
     setMobileOpen(!mobileOpen);
+  } // eslint-disable-next-line
```

Comment:
currentab and setCurrentTab are not being used anywhere and could have just been removed instead of the overly liberal use of eslint-disable